src/form.tsx

```diff
@@ -225,7 +225,7 @@ const Form = ({
                 id={id}
                 onSubmit={handleSubmit}
                 ref={formRef}
-                className={classnames(sprinkles({width: '100%'}), className)}
+                className={classnames(styles.form, className)}
```

Comment:
hmm, not sure why was this 100% width needed. The `<form>` element is 100% width by default

Reply:
Discussed offline. We are not really sure why this was added in the first place, and I'm afraid of breaking something if we remove it. After all, having that doesn't really break anything anyways